### DISCUSSION TITLE
ignore type checking for some folders

### DISCUSSION THREAD

**Discussion by FieryRMS** [microsoft/pyright#7088] [Q&A]
my file structure is like this:
```
root/
├─ src1.py
├─ sub1/
│  ├─ scr2.py
```
I want to have strict type checking for scripts in root, but not in sub1.
I tried,
1. add `pyrightconfig.json` in the root with `"typeCheckingMode": "strict"` and another one in sub1, with `"typeCheckingMode": "off"` . does not work, sub1 still has strict type checking
1. `"exclude": [ "sub1/*" ]` in root. does not work, sub1 still has strict type checking
1.  `"ignore": [ "sub1/*" ]` but that turns off all analysis. I only want to turn off only type-checking.
1. ```js
   // root pyrightconfig.json
   {
       "typeCheckingMode": "strict",
       "exclude": ["sub1/*"]
   }
    ```
    ```js
    // sub1 pyrightconfig.json
    {
        "typeCheckingMode": "off"
    }
    ```
    does not work, sub1 still has strict type checking

**Comment by erictraut** [comment]
The `strict` option is designed for cases like this.

```json
{
    "strict": ["*.py"]
}
```
